### PR TITLE
[MRG+1] Fix missing values

### PR DESCRIPTION
**Pull request by sortafreel**
Issue: https://github.com/scrapy/scrapy/issues/3804

### CODE REVIEW

**Comment by Gallaecio** [comment]
Could you please include an automated test covering your changes?

File: tests/test_loader.py [hunk 1]
```diff
@@ -709,28 +746,28 @@ def test_nested_load_item(self):
 
 
 class SelectJmesTestCase(unittest.TestCase):
-        test_list_equals = {
```
Comment (Gallaecio) [inline tests/test_loader.py]: :lipstick: I would avoid these indentation fixes, being unrelated to the actual changes.

Comment (sortafreel) [reply tests/test_loader.py]: Fixed, thanks :)

**Comment by sortafreel** [comment]
As discussed with @Gallaecio, bugs are random and not caused by this pull request code changes.

**Pull request merged**